pkgs/tools/misc/gay/test-colorizes-hello-world.nix

```diff
@@ -0,0 +1,10 @@
+{ stdenv, gay }:
+
+stdenv.mkDerivation {
+  name = "gay-test-colors-hello-world";
+  meta.timeout = 10;
+  buildCommand = ''
+    echo Hello World | ${gay}/bin/gay | cat -v | grep -v "Hello World"
```

Comment:
Correct me if I'm wrong, but it seems like this does nothing more than just testing if the binary doesn't crash? If that's the case, it might be more appropriate to use `installCheckPhase` in the main derivation.

Reply:
In retrospect, this is a fairly pointless test, so I've removed it. There's not much that can be tested (other than just comparing outputs).